DQMOffline/Muon/src/CSCOfflineMonitor.cc

```diff
@@ -1114,8 +1114,8 @@ void CSCOfflineMonitor::doPedestalNoise(edm::Handle<CSCStripDigiCollection> stri
       // for ganged ME11A
       if ((kStation == 1 && kRing == 4) && cscGeom->gangedStrips()) {
         kRing = 1;
-        if (myStrip <= 16)
-          myStrip += 64;  // no trapping for any bizarreness
+        // if (myStrip <= 16)
```

Comment:
Value of myStrip is not used (also noticed in this comment - https://github.com/cms-sw/cmssw/pull/46247/files#diff-fdf12f87d5748ce3c734bbd966b70ebde5907fbb31fdc35bfa921097f326e79eR1113)

Reply:
we can delete this too